patches/minecraft/net/minecraft/tileentity/TileEntityLockable.java.patch

```diff
@@ -0,0 +1,27 @@
+--- ../src-base/minecraft/net/minecraft/tileentity/TileEntityLockable.java
++++ ../src-work/minecraft/net/minecraft/tileentity/TileEntityLockable.java
+@@ -47,4 +47,24 @@
+     {
+         return (IChatComponent)(this.func_145818_k_() ? new ChatComponentText(this.func_70005_c_()) : new ChatComponentTranslation(this.func_70005_c_(), new Object[0]));
+     }
++
++    private net.minecraftforge.items.IItemHandler itemHandler;
++
++    public net.minecraftforge.items.IItemHandler getItemHandler(){
++        return itemHandler == null ? (itemHandler = new net.minecraftforge.items.wrapper.InvWrapper(this)) : itemHandler;
```

Comment:
I think your goal here was to allow for subclasses to make their own handlers, however, this jsut makes things derpy and doesnt specify very well that this function should CACHE the value. I would make the above field protected, and then make this a PROTECTED {no clue why its poublic} 'createItemHandler' method and have if (itemHandler == null) itemHandler = createItemHandler() in getCaps.